mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-listener.adoc

```diff
@@ -0,0 +1,156 @@
+= Listen for New Messages with MQTT Connector
+
+In these examples, you configure the Anypoint Connector for MQTT (MQTT Connector) *On New Message* source that enables you to listen for new incoming messages using one or more topic filters. Each topic filter has a specific quality of service (QoS) configured.
+
+In the following example, you configure the *On New Message* source:
+
+. In the *Mule Palette* view of Studio, select *MQTT3 > On New Message*.
+. Drag *On New Message* to the Studio canvas.
+. In the *On New Message* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. In the *MQTT3 Config* window, for *Connection*, select one of the connection types to provide to this configuration:
+
+* *MQTT3 URL Connection*
+* *MQTT3 Fail-Over Connection*
+* *MQTT3 Form Connection*
+
+[start=6]
+. Specify a unique and intuitive *Client ID* to identify an MQTT connection to a broker.
+. On the *General* tab, also specify connection information such as required libraries for the broker and advanced connection options.
+. On the *LWT* tab, optionally specify Last Will and Testament message.
+. On the *TLS/SSL* tab, optionally specify a TLS configuration.
+. On the *Advanced* tab, optionally specify a reconnection strategy.
+. Click *OK* to close the window.
+. In the *On New Message* configuration screen, in *Topics*, select *Edit inline* to list the topics that the listener subscribes to.
+. Click the plus sign (*+*) to configure a topic.
+. In the *Topic* configuration screen, set *Topic filter* that represents a single or multilevel subscription to a topic.
+. Set *QoS* to any of the following quality of service levels:
++
+* *AT_MOST_ONCE*
+* *AT_LEAST_ONCE*
+* *EXACTLY_ONCE*
+. Click *Finish*.
```

Comment:
```suggestion
. Click *Finish*:
```
...IF the graphic that follows illustrates either clicking Finish or what appears after clicking Finish.

Reply:
Globally review the sentence directly before a display element (list, table, graphic...) to ensure that it ends with a colon rather than a period IF that sentence directly introduces the element.